src/scripts/main.js

```
@@ -1 +1,85 @@
 'use strict';
+
+const body = document.querySelector('body');
+const div1 = document.createElement('div');
+const div2 = document.createElement('div');
+const div3 = document.createElement('div');
+
+div1.setAttribute('data-qa', 'notification');
+div2.setAttribute('data-qa', 'notification');
+div3.setAttribute('data-qa', 'notification');
+
+const firstPromise = new Promise((resolve, reject) => {
+  const onClick = function () {
+    resolve('First promise was resolved');
+    clearTimeout(timerId);
+  };
+
+  body.addEventListener('click', onClick);
+
+  const timerId = setTimeout(() => {
+    body.removeEventListener('click', onClick);
+    reject(Error);
+  }, 3000);
+});
+
+firstPromise
+  .then((message) => {
+    div1.textContent = message;
+    div1.classList.add('success');
+    body.append(div1);
+  })
+  .catch(() => {
+    div1.textContent = 'First promise was rejected';
+    div1.classList.add('error');
+    body.append(div1);
```

The `.catch` handler should display the message received from the promise rejection. Instead of hardcoding the message text, please accept the rejection reason as an argument to the callback (e.g., `.catch(message => ...)` ) and use that to set the text content.